src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateUtils.java

```diff
@@ -287,4 +267,19 @@ public static String substitute(String s, Map<String, String> properties, String
         }
         return s;
     }
+
+    private static List<AbstractContainerEnvVar> combineEnvVars(ContainerTemplate parent, ContainerTemplate template) {
+        List<AbstractContainerEnvVar> combinedSimpleEnvVars = new ArrayList<>();
```

Comment:
I think that the name here is misleading, right?

It seems that we are talking about all env vars not just simple, right?